app/models/product.rb

```diff
@@ -498,6 +498,7 @@ def set_initial_values
   # Try to find the best name for the new products
   def choose_default_name
     return if name.present?
+    return self.name = "blank" if (name.blank? && self.class == Product)
```

Comment:
Why would you set the name to the literal string "blank"? Also what is is `== Product` in the condition for?

Reply:
@Aquaj ,
Reason behind this when we create seeding or planting this method call multiple times, That the reason I have used "== Product" to check the Object, And I have to set "blank"  because we get this blank value on the other model, So we easy to get this value and update accordingly.

Reply:
I am afraid I still don't really get what you mean.